core/ast_node.js

```diff
@@ -230,9 +228,10 @@ Blockly.ASTNode.prototype.getParentInput = function() {
 /**
  * Get either the next editable field, or the first editable field for the given
  * input.
- * @param {!Blockly.Field} location The current location of the cursor.
+ * @param {!Blockly.Field|Blockly.Connection|Blockly.Block} location The current
```

Comment:
Missing parentheses: this says if it's a field it's non-nullable but if it's a block or connection it can be null.

Reply:
fixed